docs/usage/containerd/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Containerd Configurations
```

Comment:
```suggestion
title: containerd Configurations
```

If the title is stylized all lowercase, then it should be `containerd` instead of `Containerd`